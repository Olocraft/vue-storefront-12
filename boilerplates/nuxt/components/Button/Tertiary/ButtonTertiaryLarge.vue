<template>
  <button
    v-focus
    class="inline-flex items-center justify-center p-3 text-base font-normal bg-transparent rounded-md cursor-pointer text-primary-500 hover:bg-green-100 active:bg-green-200 disabled:bg-transparent hover:text-primary-600 active:text-primary-700 disabled:text-gray-500 disabled:opacity-50 font-body outline-violet"
    v-on="$listeners"
  >
    <slot />
  </button>
</template>

<script>
import { focus } from '../../../utils/focus-directive.js';

export default {
  name: 'ButtonTertiaryLarge',
  directives: {
    focus
  },
  props: {
    disabled: {
      type: Boolean,
      default: false
    }
  }
};
</script>
