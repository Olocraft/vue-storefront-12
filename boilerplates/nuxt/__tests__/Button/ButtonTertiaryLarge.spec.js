import { shallowMount } from '@vue/test-utils'
import ButtonTertiaryLarge from '../../components/Button/Tertiary/ButtonTertiaryLarge.vue'
let wrapper

describe('ButtonTertiaryLarge.vue', () => {
  beforeEach(() => {
    wrapper = shallowMount(ButtonTertiaryLarge)
  })
  it('render component correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
