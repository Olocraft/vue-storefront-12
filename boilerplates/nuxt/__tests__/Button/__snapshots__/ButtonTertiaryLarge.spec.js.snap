// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonTertiaryLarge.vue render component correctly 1`] = `<button class="inline-flex items-center justify-center px-6 py-3 text-base font-medium text-green-500 bg-transparent rounded-md cursor-pointer hover:bg-green-100 active:bg-green-200 disabled:bg-transparent hover:text-green-600 active:text-green-700 disabled:text-gray-500 disabled:opacity-50 font-body outline-violet"></button>`;
